File: Unit06/WS.Unit06.Example1/WS.Unit06.Example1.WebService/ICalculatorWS.cs
using System.ServiceModel;

namespace WS.Unit06.Example1.WebService
{
    [ServiceContract(Namespace = "http://ws.unit06.example1/")]
    public interface ICalculatorWS
    {
        [OperationContract]
        public double Add(double term1, double term2);
    }

}

File: Unit03/WS.Unit03.Example1/src/main/java/es/uniovi/miw/ws/unit03/example1/webservices/ICalculatorWS.java
package es.uniovi.miw.ws.unit03.example1.webservices;
import jakarta.jws.WebMethod;
import jakarta.jws.WebService;
import jakarta.jws.soap.SOAPBinding;

@WebService
@SOAPBinding(style = SOAPBinding.Style.DOCUMENT)
public interface ICalculatorWS {
    @WebMethod
    public double add(double term1, double term2);
    @WebMethod
    public double sub(double term1, double term2);
    @WebMethod
    public double mul(double term1, double term2);
    @WebMethod
    public double div(double term1, double term2) throws DividedByZeroException;
}
